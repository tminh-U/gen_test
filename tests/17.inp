6
2 7 3 20 6 17 
14
4 4
3 3
2 4
1 1
5 5
3 5
2 6
1 5
4 5
5 6
3 6
5 5
6 6
3 5
