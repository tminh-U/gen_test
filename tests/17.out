2
2
6
2
2
6
13
16
3
4
10
2
2
6
